docs/how-to.md

```diff
@@ -245,6 +245,50 @@ end
 !!!
 +++
 
+
+### How to Paginate While Maintaining Independent Contexts
+
+If you're using [hotwire](https://hotwired.dev/) ([turbo-rails](https://github.com/hotwired/turbo-rails) - if you're using Rails) when one way of maintaining independent contexts is as below:
+
+```html+erb
+  <-- movies#bad_movies -->
+  <%= turbo_frame_tag "bad_movies", src: bad_movies_path do %>    
+      <%= render "movies_table", locals: {movies: @movies}%>
+      <%== pagy_bootstrap_nav(@pagy) %>    
+  <% end %>
+
+  <-- movies#good_movies -->
+  <%= turbo_frame_tag "good_movies", src: good_movies_path  do %>    
+      <%= render "movies_table", locals: {movies: @movies}%>
+      <%== pagy_bootstrap_nav(@pagy) %>    
+  <% end %>   
+```
+
+```rb
+  # controller action 
+  def good    
+    @pagy, @movies = pagy(Movie.good, items: 5) 
+  end 
+
+  def bad    
+    @pagy, @movies = pagy(Movie.bad, items: 5)
+  end 
+```
+
+You may wish to scope your records using a search form (perhaps using [ransack](https://github.com/activerecord-hackery/ransack) etc.), to the same url, or a different url, or you may wish to use an entirely different model (actors, rather than movies): but the logic remains the same: wrap each independent context in a `turbo_frame_tag` and ensure each url returns a matching `turbo_frame_tag`.
+
+Or alternatively you may which to paginate [two different models in the same page](https://www.imaginarycloud.com/blog/how-to-paginate-ruby-on-rails-apps-with-pagy/):
+
+```rb
+def index # controller action
+  @pagy_stars, @stars = pagy(Star.all, page_param: :page_stars)
+  @pagy_nebulae, @nebulae = pagy(Nebula.all, page_param: :page_nebulae)
+end
+```
+
+Please note how a different `page_param` is specified for stars vs nebulae? This is critical if users cut and paste urls.
```

Comment:
```suggestion
Notice how a different `:page_param` is specified for stars vs nebulae. This is critical if users cut and paste urls.
```

Reply:
I had to research note vs notice verb, that I always wrongly used interchangeably. 👍🏻

Reply:
> always wrongly used interchangeably. 

It's not "wrong". You can think of it as a stylistic choice:

* Notice how I never said anything about it till now?  (this feels right)
* Note how I never said anything about it now?  (this feels "wrong")

Reply:
not according to [this](https://ell.stackexchange.com/questions/150903/what-is-the-difference-between-note-and-notice#:~:text=To%20notice%20X%20is%20to,to%20actually%20write%20X%20down)